tox.ini

```
@@ -18,12 +19,13 @@ extras = testing
 deps =
     coverage
 
-    django3.2: Django~=3.2
-    django4.0: Django~=4.0
-    django4.1: Django~=4.1
+    django3.2: Django>=3.2, <4.0
+    django4.1: Django>=4.1, <4.2
+    django4.2: Django>=4.2, <4.3
 
     wagtail4.1: wagtail>=4.1, <4.2
     wagtail4.2: wagtail>=4.2, <5.0
+    wagtail5.0: wagtail>=5.0, <6.0
```

```suggestion
    wagtail5.0: wagtail>=5.0, <5.1
```

The next release is planned as 5.1 as far as I am aware